docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -145,10 +145,10 @@ Border styles.
 
 | Property  | Type   |  Props  |
 | ---       | ---    |---   |
-| color | string |  |
+| color | undefined |  |
```

Comment:
These `undefined` types are the result of running `npm run docs:build`. It looks like these changes need to match the result of this script when run on GitHub, so I've pushed these changes for now. However, this should probably be improved, as `undefined` isn't that helpful.

I'll spin up a follow-up PR if I figure this out!

Reply:
There isn't anything special going on with the docgen—just reading in the json file and formatting a few things as markdown. So it shouldn't be too hard to update the docgen to aggregate the types in the `oneOf` list which would also fix the `radius` property.

Eventually it would be nice to generate the schema from a higher-level source that is more suited for docgen. There's a [discussion](https://github.com/Automattic/dotorg-devrel/discussions/421) open about that, but I'd like to see the quick fix before tackling the larger problem.

Reply:
Thanks @ajlende, that's super helpful context!

Reply:
Created a PR for this here: https://github.com/WordPress/gutenberg/pull/46375